cmd/aida-vm-sdb/run_substate_test.go

```diff
@@ -214,12 +220,18 @@ func TestVmSdb_Substate_ValidationDoesNotFailOnValidTransaction(t *testing.T) {
 	ctrl := gomock.NewController(t)
 	provider := executor.NewMockProvider[txcontext.TxContext](ctrl)
 	db := state.NewMockStateDB(ctrl)
+
+	chainConfig, err := utils.GetChainConfig(utils.EthTestsChainID)
+	if err != nil {
+		t.Fatalf("cannot get chain config: %v", err)
+	}
 	cfg := &utils.Config{
```

Comment:
I wonder if we should implement a new `NewConfig` for testing or implement `cfg.SetChainConfig()`? It would be nicer if `ChainCfg` can be set from `ChainID`. I see  quite a bit of code repetition with declaring `chainCfg` first then create a new Config.